googleapis_auth/lib/src/auth_http_utils.dart

```diff
@@ -94,8 +94,10 @@ class AutoRefreshingClient extends AutoRefreshDelegatingClient {
     this.credentials, {
     bool closeUnderlyingClient = true,
     this.quotaProject,
-  })  : assert(credentials.accessToken.type == 'Bearer'),
-        assert(credentials.refreshToken != null),
+  })  : assert(credentials.accessToken.type == 'Bearer',
```

Comment:
These don't offer much. When asserts fail you see the code in the assert – and in each the assert is pretty clear.

Reply:
Yeah...tis why this assert is not recommended broadly!